### DISCUSSION TITLE
Welcome to monet Discussions!

### DISCUSSION THREAD

**Discussion by bbakernoaa** [noaa-oar-arl/MONET#96] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by praful-dodda** [comment]
Can someone comment why modules like "sat" and "obs" are removed? or Are there any substitutions? Thanks in advance

**Comment by bbakernoaa** [comment]
This is a good question! monet has been split into two repositories - monet
and monetio.  This was done to simplify requirements and tools for the
respective operations.

You can checkout the new repository at
https://github.com/noaa-oar-arl/monetio

Note that monetio is only the I/O component of monet.
